hircluster.c

```diff
@@ -195,9 +192,8 @@ static void __redisClusterSetError(redisClusterContext *cc, int type,
                                    const char *str) {
     size_t len;
 
-    if (cc == NULL) {
+    if (cc == NULL)
         return;
-    }
```

Comment:
I suggest not doing this kind of changes, since it clutters the git history and diverges unnecessarily from hiredis-vip.

Or is it required by some code formatting checker?

Most projects I've been following would reject changes like this, even when there may be a preference on with/without braces when adding new code.

Reply:
I was aiming for the same look-and-feel as hiredis, which I think would be good.
An option would be to do this kind of changes incrementally when improving functions, and then instead state that as a goal in CONTRIBUTING.md ?

Reply:
OK, good then. We can do everything at one go to set a standard. As long as we don't do ping pong changes back and forth. We can put some guidelines in CONRIBUTING.md if you want. We could perhaps also write that you shouldn't do unrelated formatting changes in the same commit as something else though.

Reply:
I removed the commit regarding compact if-cases, you are right the commit will clutter the history.